### PR TITLE
DX-2023 Update voice code snippets

### PR DESCRIPTION
**Pull request by hamermike**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-task-update-voice-snippets.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — DISMISSED** [review]

File: sdks/python.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ The Python SDK(s) are available via [PyPi](https://pypi.org/) & Github
 | 7.0.0 | Renamed `CallEngineModifyConferenceRequest` to `ApiModifyConferenceRequest`, and removed `from` and `digits` from `TwoFactorVerifyRequestSchema` |
 | 8.0.0 | Added get messages function, and updated the `body` parameter in the create message function to be required |
 | 9.0.0 | Updated the MFA error bodies and added message priority |
+| 10.0.0 | Updated voice, messaging, and MFA objects as well as corrected WebRTC `participantId` and `sessionId` parameter ordering in a number of requests. |
+| 10.1.0 | Add transfer BXML verb for WebRTC. |
+| 11.0.0 | Add voice `voiceCallId` to the transfer BXML verb for WebRTC. |
 
 
 ## Download & Install
```
Comment (ajrice6713) [inline sdks/python.md:35]: Line 52 should be 

`from bandwidth.voice.models.create_call_request import CreateCallRequest`

Comment (ajrice6713) [reply sdks/python.md:35]: I bet you this has changed on the other languages as well

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: sdks/python.md (hunk 1 above)
Comment (ajrice6713) [reply sdks/python.md:35]: I bet this has changed on the other languages as well

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-task-update-voice-snippets.s3-website-us-east-1.amazonaws.com/

**Review by ajrice6713 — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-task-update-voice-snippets.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-task-update-voice-snippets.s3-website-us-east-1.amazonaws.com/

**Pull request merged**